app/uk/gov/hmrc/apidefinition/controllers/APIDefinitionController.scala

```diff
@@ -32,6 +32,7 @@ import uk.gov.hmrc.http.{HeaderCarrier, UnauthorizedException}
 import uk.gov.hmrc.play.bootstrap.controller.BackendController
 
 import scala.concurrent.{ExecutionContext, Future}
+import _root_.uk.gov.hmrc.apidefinition.models.APICategory
```

Comment:
Remove any _root_ type imports and instead import from correct place - in this case:

`import uk.gov.hmrc.apidefinition.models.APICategory`

Reply:
I have resolved this